packages/client/src/createClient.ts

```diff
@@ -131,6 +144,64 @@ export function createClientInternal(
         o,
       ) as (T extends QueryDefinition<any> ? QuerySignatureFromDef<T>
         : never) as any;
+    } else if (o.type === "experiment") {
+      switch (o.name) {
+        case __EXPERIMENTAL__NOT_SUPPORTED_YET__getBulkLinks.name:
+          return {
+            invoke: createBulkLinksAsyncIterFactory(
+              clientCtx,
+            ),
+          } as any;
+        case __EXPERIMENTAL__NOT_SUPPORTED_YET__preexistingObjectSet.name:
```

Comment:
Do these as any's have any effect on the types returned from these?

Reply:
not really, these are all just moved from their old locations so the logic doesnt change and the interface for `Client` actually dictates what the consumer sees.

I went down a rabbit hole of trying to make these super strict but it was getting out of hand and realized that it would be better to follow up later with trying to prevent the future issues.